server/src/main/java/org/apache/iotdb/db/query/control/TracingManager.java

```diff
@@ -198,8 +198,46 @@ public void writeEndTime(long queryId) throws IOException {
     writer.flush();
   }
 
-  public void close() throws IOException {
-    writer.close();
+  public void close() {
+    try {
+      writer.close();
+    } catch (IOException e) {
+      logger.error("Meeting error while Close the tracing log stream : {}", e.getMessage());
+    }
+  }
+
+  public boolean getWriterStatus() {
+    try {
+      writer.flush();
+      return true;
+    } catch (IOException e) {
+      return false;
+    }
+  }
+
+  public void openTracingWriteStream() {
+    File tracingDir =
+        SystemFileFactory.INSTANCE.getFile(
+            IoTDBDescriptor.getInstance().getConfig().getTracingDir());
+    if (!tracingDir.exists()) {
+      if (tracingDir.mkdirs()) {
+        logger.info("create performance folder {}.", tracingDir);
+      } else {
+        logger.info("create performance folder {} failed.", tracingDir);
+      }
+    }
```

Comment:
Hi, could you abstract the logic in constructor of TracingManager and this as a single method?

Reply:
no problem